Interfaces/Preferences.xib

```diff
@@ -1501,23 +1502,60 @@ Warning: If Sequel Ace and the server do not have at least one cipher suite in c
                         <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                     </textFieldCell>
                 </textField>
+                <textField verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="5Vs-H5-Nca">
+                    <rect key="frame" x="18" y="220" width="154" height="21"/>
+                    <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" editable="YES" sendsActionOnEndEditing="YES" alignment="right" title="Known Hosts:" id="dyI-8y-FNW">
+                        <font key="font" metaFont="system"/>
+                        <color key="textColor" name="controlTextColor" catalog="System" colorSpace="catalog"/>
+                        <color key="backgroundColor" name="textBackgroundColor" catalog="System" colorSpace="catalog"/>
+                    </textFieldCell>
+                </textField>
+                <popUpButton verticalHuggingPriority="750" tag="2" translatesAutoresizingMaskIntoConstraints="NO" id="l1q-C8-ytL">
+                    <rect key="frame" x="177" y="216" width="347" height="27"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="22" id="tk9-kk-iZA"/>
+                    </constraints>
+                    <popUpButtonCell key="cell" type="push" title="Item 1" bezelStyle="rounded" alignment="left" lineBreakMode="truncatingTail" state="on" borderStyle="borderAndBezel" imageScaling="proportionallyDown" inset="2" selectedItem="geP-YV-B0g" id="hmJ-F4-sdq">
+                        <behavior key="behavior" lightByBackground="YES" lightByGray="YES"/>
+                        <font key="font" metaFont="menu"/>
+                        <menu key="menu" id="jFV-QJ-gOA">
+                            <items>
+                                <menuItem title="Item 1" state="on" id="geP-YV-B0g"/>
+                                <menuItem title="Item 2" id="IYc-iy-xas"/>
+                                <menuItem title="Item 3" id="vG7-NQ-ak6"/>
+                            </items>
+                        </menu>
+                    </popUpButtonCell>
+                    <connections>
+                        <action selector="updateKnownHostsConfig:" target="2146" id="U2t-D8-vr8"/>
+                        <binding destination="117" name="selectedTag" keyPath="values.known_hosts" id="1uC-TG-LLr">
+                            <dictionary key="options">
+                                <bool key="NSAllowsEditingMultipleValuesSelection" value="NO"/>
+                                <string key="NSNullPlaceholder">Sequel Ace default</string>
```

Comment:
Could we add an option "use known hosts from ssh config (ADVANCED)" or something, which would require the user to choose a known hosts file in their ssh config file? Would that fix that special use case?

Reply:
i think so